spec/flipper/adapters/active_record_spec.rb

```diff
@@ -215,18 +215,27 @@
         end
       end
 
-      it "works when table doesn't exist" do
```

Comment:
I noticed that this test wasn't actually testing the table not existing, but the fact that it's not even connected to the DB. Without the `silence`, you can see the message:

```
No connection pool for 'ActiveRecord::Base' found.. You likely need to run `rails g flipper:active_record` and/or `rails db:migrate`.
```